CHANGES.md

```diff
@@ -20,7 +20,7 @@ To be released.
     `IBlockPolicy<T>.ValidateNextBlock(IReadOnlyList<Block<T>>, Block<T>)`
     method.  [[#210]]
  -  Removed `IBlockPolicy<T>.ValidateBlocks()` method.  [[#210]]
- -  Added `IBlockPolicyExtension.ValidateBlocks<T>(IBlockPolicy<T>,
+ -  Added `BlockPolicyExtension.ValidateBlocks<T>(IBlockPolicy<T>,
     IReadOnlyList<Block<T>>, DateTimeOffset)` method.  [[#210]]
```

Comment:
As it is actually not an `interface` change, it should go to “added interfaces.”